Projects/TravelCertificate/Sources/Store/TravelInsuranceStore.swift

```diff
@@ -120,24 +122,13 @@ final class TravelInsuranceStore: LoadingStateStore<
         case let .setEmail(value):
             newState.travelInsuranceModel?.email = value
         case .toogleMyselfAsInsured:
-            newState.travelInsuranceModel?.isPolicyHolderIncluded.toggle()
+            newState.travelInsuranceModel?.isPolicyHolderIncluded = true
         case let .setPolicyCoInsured(data):
             let indexOfUser = newState.travelInsuranceModel?.policyCoinsuredPersons
```

Comment:
And here after updating 
```
case setPolicyCoInsured(PolicyCoinsuredPersonModel)
to
case setPolicyMembers([PolicyCoinsuredPersonModel])
```
mentioned bellow.

Just set newState.travelInsuranceModel?.policyCoinsuredPersons with the new value excluding owner of the contract.  Other check is if the contract owner  is included and set that value to true/ false